app/javascript/app/document-capture/components/submission-pending.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Image from './image';
+
+function SubmissionPending({ onComplete }) {
+  useEffect(() => {
+    const timeoutId = setTimeout(onComplete, 2000);
```

Comment:
Same note about configurable JS timeouts: https://github.com/18F/identity-idp/pull/3964#discussion_r459092935, since this is multiple places sounds like a better for for a later PR?

Reply:
> Same note about configurable JS timeouts: [#3964 (comment)](https://github.com/18F/identity-idp/pull/3964#discussion_r459092935), since this is multiple places sounds like a better for for a later PR?

The same response as https://github.com/18F/identity-idp/pull/3964#discussion_r459099922 applies here, which is: This timeout is arbitrary, and eventually would leverage the progression from #3964 instead.

But as noted in the original comment, it does make for some awkwardness in that: This pull request introduces test utilities around managing timeouts, where the timeouts it's helping to manage are fully intended to be removed in the future.

🤷  I'm not overly concerned about removing the utility when the time comes. The Sinon upgrade will be nice to have regardless.